### DISCUSSION TITLE
Aging model in SPM/SPMe

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#3288] [Q&A]
Can we implement SEI and lithium plating model options in SPM or SPMe models in PyBaMM, and what are the equations for plating and sei implemented in SPM or SPMe model if it can be implemented? 
It would be great if there is some reference for the equations for plating and SEI in SPM model.

Thanks in advance.

**Comment by wass-cloud** [comment]
Yes you can, any equations can be found in Okane2022. My work is very similar, so let me know if you get a thermal model working, since that can prove difficult (at least for me).

link to Okane2022: http://dx.doi.org/10.1039/D2CP00417H

**Reply by kumaryash7** [reply]
So the same SEI and plating equations are implemented in all the three DFN, SPM and SPMe models?

**Reply by brosaplanella** [reply]
With slight differences, as DFN and SPM are different models, but yes.

**Reply by kumaryash7** [reply]
What are the differences that are incorporated? Can you refer something?
Thanks in advance

**Reply by brosaplanella** [reply]
Basically that each model uses their own definitions of potential and concentration, so they are consistent.
